src/pages/booking.js

```diff
@@ -0,0 +1,77 @@
+import styles from "@/styles/booking.module.scss";
+import { useState } from "react";
+
+export default function Booking()
+{
+    // Mock data to be removed when real data is fixed
+    let title = "Blade"
+    let date = "2023-06-24"
+    let time = "14:30"
+    let salong = "Alcazar"
+    let lenght = "120"
+    let imageURL = "https://m.media-amazon.com/images/M/MV5BOTk2NDNjZWQtMGY0Mi00YTY2LWE5MzctMGRhZmNlYzljYTg5XkEyXkFqcGdeQXVyMTAyNjg4NjE0._V1_SX300.jpg"
+
+    const [adultPrice, setAdultPrice] = useState(150);
```

Comment:
I fixed the Link on the movie page.

To access the information, use the following:

```javascript
export function getServerSideProps(context) {
  console.log(context.query) 
  return {
      props: { 
         title: context.query.title,
         length: context.query.length,
         showtimeId: context.query.showtimeId // etc...
      }
  }
}```